### DISCUSSION TITLE
How can I change the QR code on the start game page?

### DISCUSSION THREAD

**Discussion by jamesnahon** [mawoka-myblock/ClassQuiz#413] [Q&A]
hi there,
i've got classquiz mostly running on an AWS EC2.  I've built my quiz and tested it, everything works really well!

the problem i have is that the QR code is linking to classquiz.de, not the address i'm hosting at.

How can i update this please?

**Comment by mawoka-myblock** [comment]
You haven't changed the rooot address of the API container
